drivers/eks/eks_driver.go

```diff
@@ -194,6 +199,16 @@ func (d *Driver) GetDriverCreateOptions(ctx context.Context) (*types.DriverFlags
 			DefaultBool: true,
 		},
 	}
+	// If this is used it will overwrite the default values in the template and use what is given here
+	// Expectation: leave this as "" otherwise show what was there previously with a warning to be careful
+	//	modifying it
+	driverFlag.Options["user-data"] = &types.Flag{
+		Type:  types.StringType,
+		Usage: "Pass user-data to the nodes to perform automated configuration tasks",
+		Default: &types.Default{
+			DefaultString: "",
```

Comment:
Strings should default to`""` (empty string) anyway, right?  I think this is unnecessary unless there is a compelling reason I'm not seeing.

Reply:
Unfortunately not, it is set to nil if it is not initialized which becomes an issue in testing where GET requests return cluster objects with a nil subnet.